docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -19,30 +19,36 @@ YugabyteDB Anywhere (YBA) high availability (HA) is an active-standby model for
 
 Each HA cluster includes a single active YBA instance and at least one standby YBA instance, configured as follows:
 
-- The active instance runs normally, but also pushes out backups of its state to all of the standby instances in the HA cluster.
+- The active instance runs normally, but also pushes out backups of its state to all of the standby instances in the HA cluster at a configurable frequency (no more than once per minute).
+
+    The active instance also creates and sends one-off backups to standby instances whenever a task completes (such as creating a new universe).
+
 - A standby instance is completely passive while in standby mode and can't be used for managing or monitoring clusters until you manually promote it to active.
```

Comment:
Standby can be used for monitoring Prometheus data so probably want to remove that word. It can't be used for any YBA actions though.